ipaserver/plugins/host.py

```diff
@@ -27,9 +27,10 @@
 import six
```

Comment:
I did notice this in user.py plugin but forgot to check with the other two. I'll fix it, thanks.